tests/integration/targets/ec2_instance/roles/ec2_instance/tasks/termination_protection_wrapper.yml

```diff
@@ -0,0 +1,31 @@
+---
+- include_role:
+    name: 'setup_remote_tmp_dir'
+
+- set_fact:
+    virtualenv: "{{ remote_tmp_dir }}/virtualenv"
+    virtualenv_command: "{{ ansible_python_interpreter }} -m virtualenv"
+
+- set_fact:
+    virtualenv_interpreter: "{{ virtualenv }}/bin/python"
+
+- pip:
+    name: "virtualenv"
+
+- pip:
+    name:
+      - awscli<=1.18.159
+      - botocore<1.19.0,>=1.13.3
```

Comment:
What's the problem with botocore 1.19 ?

Reply:
It's what we currently constrain the tests to tests/integration/constraints.txt
I didn't want to deviate too far from what we're testing everything else against since this isn't a boto rev test, just get the version of awscli that would work with those constraints.